lib/search-field/index.jsx

```diff
@@ -36,58 +38,68 @@ export class SearchField extends Component {
     }
   }
 
+  doSearch = query => {
+    this.setState({ query, searchSelected: true });
+    this.props.onSearch(query);
```

Comment:
It appears as though we may have been using `query` because we were splitting the data flow of the search bar text with the debounced copy. Now that we're taking the debounce out I question whether we need to continue to split the data stream. We might be able to eliminate all of the `this.state.query` and rely on the `filter` provided by Redux.

The `this.props.onSearch` function doesn't appear to do anything except store this `filter` or `query` so I think if we add `state.filter` in `mapStateToProps` we can rely on it instead of `this.state.query`

Reply:
Tabling this for now (see [below](https://github.com/Automattic/simplenote-electron/pull/1643#issuecomment-543523665)) but will revisit if we do decide to remove the filtering.